nautobot/docs/rest-api/filtering.md

```diff
@@ -52,8 +52,16 @@ To filter results by a custom field value, prepend `cf_` to the custom field nam
 GET /api/dcim/sites/?cf_foo=123
 ```
 
+Multi-value filtering is available for all field types and lookup expressions. When multiple values are supplied, an `or` lookup will be performed by the query. For example, the following query will return only sites where a custom field named `foo` is equal to 123 _or_ 456:
+
+```no-highlight
+GET /api/dcim/sites/?cf_foo=123&cf_foo=456
+```
+
 Custom fields can be mixed with built-in fields to further narrow results. When creating a custom string field, the type of filtering selected (loose versus exact) determines whether partial or full matching is used.
 
+Custom fields can use the lookup expressions listed in the next section by prepending `cf_` to the custom field name followed by the required lookup type (see below).
```

Comment:
Hmm, this may be a gap/flaw with the changes in #2136 - after that PR, in API version 1.4+ the REST API representation of custom fields uses the `slug` rather than the `name` of the fields, i.e.:

```
"custom_fields": {
    "my_custom_field_slug": "value",
    "another_custom_field_slug": "value2"
}
```

versus the pre-1.4 API behavior of:

```
"custom_fields": {
    "My custom field!": "value",
    "Some / other custom field?": "value2"
}
```

So it would probably be better if the REST API filters in API version 1.4+ also used `slug` instead of `name`... but we don't have a way to do versioned FilterSets, so that would be a breaking change to existing filters. Probably the best we can do in 1.x is call out that regardless of REST API version the filtering of custom fields always uses the field name rather than the slug for now?